frontend/src/components/Comment/SCSS/EditComment.scss

```diff
@@ -0,0 +1,75 @@
+.editButton {
+  flex-grow: 2;
```

Comment:
As mentioned in the review for #39, I would like these to be consistent, at the very least within the same file. Consider hyphenating. This applies to all class names in the file.